locales/en-us.json

```diff
@@ -191,7 +191,7 @@
     "startrequestcta": "Click the button below to start the instructor verification process.",
     "restartrequest": "Restart Verification Request",
     "biourl": "Bio URL",
-    "biourl_desc": "To verify instructor status you must provide a link to a web page showing your faculty status. Links to your institution's home page are NOT sufficient. A URL that shows that you are an instructor is needed. Pages must be publicly accessible (cannot require login).",
+    "biourl_desc": "To verify instructor status you please provide a link to a web page showing your faculty status. Links to your institution's home page are NOT sufficient. A URL that shows that you are an instructor is needed. Pages must be publicly accessible (cannot require login).",
```

Comment:
"To verify instructor status you please" -> "To verify instructor status, please"